packages/node_modules/@webex/plugin-memberships/test/integration/spec/memberships.js

```diff
@@ -395,6 +451,63 @@ describe('plugin-memberships', function () {
       });
     });
 
+    describe('#updateLastSeen()', () => {
+      let spark1, actor1, message, room;
+
+      before(() => {
+        // Get another SDK instance for the second user
+        // so that one user can send the message
+        // and another can mark it as read
+        spark1 = new CiscoSpark({credentials: user1.token});
```

Comment:
If you need two different users and two different `spark`s, you can attach the `spark` to the user. 

```js
// where spock is the user
spock.spark = new CiscoSpark({...
```

Reply:
Now implemented as you suggested